receiver/iisreceiver/metadata.yaml

```diff
@@ -115,3 +129,90 @@ metrics:
     gauge:
       value_type: int
     enabled: true
+  iis.site.request.count:
+    description: Total number of requests of a given type.
+    unit: "{requests}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+    attributes: [ request ]
+  iis.app_pool.request.rejected:
+    description: Total number of requests rejected.
+    unit: "{requests}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  iis.app_pool.request.queue.count:
+    description: Current number of requests in the queue.
+    unit: "{requests}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  iis.app_pool.request.queue.age.max:
+    description: Age of oldest request in the queue.
+    unit: ms
+    gauge:
+      value_type: int
+    enabled: true
+  iis.site.network.file.count:
+    description: Number of transmitted files.
+    unit: "{files}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+    attributes: [ direction ]
+  iis.site.network.blocked:
+    description: Number of bytes blocked due to bandwidth throttling.
+    unit: By
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  iis.site.network.io:
+    description: Total amount of bytes sent and received.
+    unit: By
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+    attributes: [ direction ]
+  iis.site.connection.attempt.count:
+    description: Total number of attempts to connect to the server.
+    unit: "{attempts}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  iis.site.connection.active:
+    description: Number of active connections.
+    unit: "{connections}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  iis.site.connection.anonymous:
+    description: Number of connections established anonymously.
+    unit: "{connections}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  iis.site.uptime:
+    description: The amount of time the server has been up.
+    unit: s
+    gauge:
+      value_type: int
```

Comment:
Why would we add a version of metrics we already have, when we have resource attributes as context?

Reply:
The main reason I did this was to not introduce breaking behavior; but maybe that's a misstep, so I'd like your opinion:

1. Should we still emit the original metrics (totals) without the new resource attributes?
2. If we DO emit the original metrics in addition to the new ones with the resource attributes, will that make this a breaking change?

My gut feeling is that doing 1 violates some rule, so a breaking change is necessary here if we don't want new metrics (which is fine, was just trying to avoid that).

Reply:
It's a valid concern. I think unfortunately we're going to end up with a breaking change one way or another because we really should not keep both sets of metrics. The component is in beta status so technically breaking changes to the emitted metrics are still allowed. So the question is how carefully we need to manage the change. We can have both sets for a while and enable/disable them by default via feature flags. Or, in my opinion based on having not seen anyone else interact with this component, I think it's reasonable to just make the change and be clear about it in the release notes. Perhaps though you have more insight into how painful the change would be.

Reply:
I don't think the change will actually be that painful; I think just biting the bullet and breaking up the existing metrics, and dropping the totals is the right way to go about it. I'll ping you when I make that change.

Reply:
@djaglowski I've changed the behavior, as described above ^ I think it's ready for another pass on it.